packages/next/build/webpack-config.ts

```diff
@@ -634,6 +634,11 @@ export default async function getBaseWebpackConfig(
     loggedIgnoredCompilerOptions = true
   }
 
+  if (!useSWCLoader && config.experimental.fontLoaders) {
+    Log.error('`experimental.fontLoaders` requires SWC but found Babel config.')
```

Comment:
```suggestion
    Log.error(`"experimental.fontLoaders" is enabled which requires SWC although Babel is being used due to custom babel config being present "${path.relative(dir, babelConfigFile)}". See more info here: https://nextjs.org/docs/messages/babel-font-loader-conflict`)
```
Can we add an error document explaining they either need to disable `experimental.fontLoaders` or remove their custom babel config?